src/plugins/data/common/field_formats/content_types/html_content_type.ts

```diff
@@ -35,13 +35,13 @@ const getConvertFn = (
       : getHighlightHtml(formatted, hit.highlight[field.name]);
   };
 
-  return (fieldFormatConvert[CONTEXT_TYPE] || fallbackHtml) as HtmlConventTypeConvert;
+  return (fieldFormatConvert[HTML_CONTEXT_TYPE] || fallbackHtml) as HtmlConventTypeConvert;
 };
 
 export const setup = (
   format: IFieldFormat,
-  fieldFormatConvert: FieldFormatConvert
-): FieldFormatConvert => {
+  fieldFormatConvert: Partial<FieldFormatConvert>
+): HtmlConventTypeConvert => {
```

Comment:
`HtmlConventTypeConvert ` .. is this a typo ?

Reply:
👍